govcd/vm_test.go

```diff
@@ -880,3 +880,30 @@ func (vcd *TestVCD) Test_VMPowerOnPowerOff(check *C) {
 	check.Assert(err, IsNil)
 	check.Assert(vmStatus, Equals, "POWERED_OFF")
 }
+
+// Test gathering VM virtual hardware items
+func (vcd *TestVCD) Test_GetVirtualHardwareSection(check *C) {
```

Comment:
Thank you for adding test. Could you improve test by creating VM with some things in VirtualHardwareSection and verify they are returned. As current tests only verify function call.

Reply:
Changed up the test a bit here and added in some `VirtualHardwareItem` data and validated that it returns the equal data.

Reply:
Thank you